docs/en/Running-Alluxio-On-Docker.md

```diff
@@ -57,12 +75,22 @@ $ docker run -d --net=host alluxio master
 
 We need to tell the worker where to find the master. Set the `ALLUXIO_MASTER_HOSTNAME`
 environment variable to your machine's hostname when launching the worker Docker container.
+To enable short-circuit reads, share the ramdisk with `-v /mnt/ramdisk:/mnt/ramdisk`, and
+specify its location and size to the worker. `-v /mnt/ramdisk:/mnt/ramdisk` will mount the
+`/mnt/ramdisk` path on the host machine to the `/mnt/ramdisk` path in the worker container.
+This way, the data written by the Alluxio worker can be directly accessed from outside the
+container.
 
 ```bash
-$ docker run -d --net=host -e ALLUXIO_MASTER_HOSTNAME=${INSTANCE_HOSTNAME} alluxio worker
+$ docker run -d \
+           -v /mnt/ramdisk:/mnt/ramdisk \
```

Comment:
Is there instructions for the client docker container? Any client container also need to use this volume mounting, right?